common.blocks/checkbox-group/checkbox-group.bemhtml

```diff
@@ -29,6 +29,7 @@ block('checkbox-group')(
                     val : option.val,
                     text : option.text,
                     title : option.title,
+                    tabIndex : option.tabIndex || ctx.tabIndex,
```

Comment:
Looks like there is no need to have separate tabindex in each option. So `ctx.tabIndex` should be enough.